src/commands/tenmans.ts

```diff
@@ -134,7 +136,63 @@ class TenmansCloseSubcommand extends MessageExecutable<CommandInteraction> {
 
     // Teardown - clear current queue
     tenmansQueue = []
-    await activeTenmansMessage.delete();
+    await activeTenmansMessage?.delete();
+  }
+}
+
+class TenmansVoteSubcommand extends RegisteredUserExecutable<CommandInteraction> {
+  async afterUserExecute() {
+    if (activeTenmansMessage) {
+      this.interaction.reply({
+        content: "You should have been looking more closely. There's already a queue - use that one instead!",
+        ephemeral: true
+      });
+
+      return;
+    }
+
+    const queueChannel = botConfig.queueMsgChannel as TextChannel;
+    if (!queueChannel) {
+      this.interaction.reply({
+        content:
+          "No queue message channel configured. Please set channel id using '/config defaultChannel'.",
```

Comment:
If end users have the power to run this command, I don't think this error message will help since they can't run the config command.

Reply:
I went ahead and modified the command using a template string.